test/PostageStamp.test.ts

```diff
@@ -88,13 +91,18 @@ describe('PostageStamp', function () {
     let postageStampStamper: Contract, token: Contract, priceOracle: Contract;
     let batch: Batch;
     let batchSize: number, transferAmount: number;
-    const price0 = 1024;
-    let setPrice0Block: number;
+    let minimumPrice: number;
+    let price0: number;
+    let initialBlock: number;
```

Comment:
why was the old name not good enough? it was really clear what was it about, now it is not that straightforward.

Reply:
hmm ok, probably right. will put it back.